parametric/README.md

```diff
@@ -138,6 +138,13 @@ To run the Python tests "locally" push your code to a branch and then specify ``
 PYTHON_DDTRACE_PACKAGE=git+https://github.com/Datadog/dd-trace-py@1.x ./run.sh ...
 ```
 
+#### NodeJS
+
+There is two ways for running the NodeJS tests with a custom tracer:
+- Place an NPM package in the folder ``apps/nodejs/npm`` and then set the enviornment variable ``NODEJS_DDTRACE_MODULE``
```

Comment:
```suggestion
- Place the ddtrace NPM package in the folder ``apps/nodejs/npm`` and then set the enviornment variable ``NODEJS_DDTRACE_MODULE``
```
?

Reply:
can we maybe have a one liner command for how one can do this (given that the repo is cloned locally).

Reply:
Yes. I will add an example for each case. Thanks for your suggestions.